asserts/pool.go

```diff
@@ -273,6 +362,52 @@ func (p *Pool) phase(ph poolPhase) error {
 	return nil
 }
 
+// AddSequenceToUpdate adds the assertion referenced by toUpdate and all its
```

Comment:
we should add to Pool own doc comment something like:

AddUnresolvedSequence and AddSequenceToUpdate exist parallel to AddUnresolved/AddToUpdate to handle sequence-forming assertions,  which cannot be used with the latter.